packages/office-ui-fabric-react/src/components/ActivityItem/ActivityItem.Props.ts

```diff
@@ -58,6 +68,11 @@ export interface IActivityItemStyles {
   activityPersona?: IStyle;
 
   /**
+   * Styles applied to the activity's description.
+   */
+  activityText?: IStyle;
```

Comment:
Check convention, but i'm pretty sure any style prop needs to be xxxXxxStyle or something. If I saw activityText I'd think it were a string.